sources/api/apiclient/README.md

```diff
@@ -75,6 +75,53 @@ You can use JSON form to set it:
 apiclient set --json '{"motd": "42"}'
 ```
 
+### Network configure
+
+This allows you to configure network settings by providing a net.toml file which netdog uses for network configuration.
+The configuration is written to `/.bottlerocket/net.toml` and will be validated and applied at the next boot by netdog.
```

Comment:
Similar to what Arnaldo mentioned, I might drop references to `net.toml` from the client configuration, and just mention that it will be applied at next boot. Protects us from stale documentation if the underlying implementation changes later.

Reply:
Updated. I think it makes sense to not mention the exact path it will be written to. But mentioning `net.toml` and its format is probably needed. See the section below (#### Configuration format).

The user needs to know how to compose the `net.toml` as the source.